pkg/pool-weighted/test/foundry/WeightedMath.t.sol

```diff
@@ -0,0 +1,98 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import { Test } from "forge-std/Test.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
+
+import "../../contracts/WeightedMath.sol";
+
+contract WeightedMathTest is Test {
+    using FixedPoint for uint256;
+
+    function testJoinSwaps(
+        uint256[20] memory balancesFixed,
+        uint256[20] memory normalizedWeightsFixed,
```

Comment:
We could go higher here, though I guess it doesn't quite matter in this test.

Reply:
Pushing too high here results in too many rejected runs due to minimum weight constraints.